### PR TITLE
change paas header implementation to be more similar to core

### LINKED ISSUE #2814: PAAS user column on master slightly broken
On the S1 user profile page, the header gets partially covered by adhocracy content. Probably caused by #2805.

paas-design kaputt

### PR DESCRIPTION
**Pull request by xi**
Fixes #2814 

Note that some spacings changed. I did not want to get into the details.

### CODE REVIEW

File: src/s1/s1/static/stylesheets/scss/components/_s1_header.scss [hunk 1]
```diff
@@ -13,38 +10,6 @@
     }
 }
 
-.s1-header {
-    margin-top: rem(50px);
-    margin-bottom: rem(10px);
-
-    position: relative;
-}
```
Comment (der-john) [inline src/s1/s1/static/stylesheets/scss/components/_s1_header.scss]: Why was this removed? It's used in `src/s1/s1/static/js/Packages/S1/Context/header.html`.

**Comment by der-john** [comment]
nice! looks good & works.

**Comment by xi** [comment]
@der-john I think you forgot to revert 	f11c524
